sdks/java/container/common.gradle

```diff
@@ -60,8 +59,12 @@ task copySdkHarnessLauncher(type: Copy) {
     into "build/target"
 }
 
-task copyPulledLicenses(type: Copy) {
-    from configurations.pulledLicenses
+task copyJavaThirdPartyLicenses(type: Copy) {
+    dependsOn ':sdks:java:container:pullLicenses'
+    from("${project(':sdks:java:container').buildDir}/build/target") {
+        include 'third_party_licenses/*'
+        include 'java_third_party_licenses/*'
```

Comment:
I'm not sure java_third_party_licenses actually gets used by our [Dockerfile](https://github.com/apache/beam/blob/master/sdks/java/container/Dockerfile) atm. I'm also not sure if there is a cleaner way to reference the parent directory build target.

Reply:
[license_script.sh](https://github.com/apache/beam/blob/master/sdks/java/container/license_scripts/license_script.sh) copies everything from `build/target/java_third_party_licenses` into `build/target/third_party_licenses`, so IIUC we shouldn't copy `java_third_party_licenses` again here.